doc: add a note to update west

ru-fu opened the pull request:
When switching from the 1.2 release to the master branch,
users get an error because west must be updated.
Add a note to point them to the solution.

Signed-off-by: Ruth Fuchss <ruth.fuchss@nordicsemi.no>

carlescufi commented:
You might also want to add a link to: 
https://docs.zephyrproject.org/latest/guides/west/troubleshooting.html#error-unexpected-keyword-argument-requires-workspace

ru-fu commented:
> 
> 
> The change looks good, but it should also be noted that you could get the error even _before_ reaching this step.
> 
> e.g. when issuing the command 'west init -m https://github.com/nrfconnect/sdk-nrf' you will get an error if you do not have the latest version of 'west'. (since releases > tag 1.2 requires newer west tool)

But we're in the installation instructions here - is it really a use case that customers install the NCS manually but don't clone the repos right after? Or that they clone the repos again for working on a different version instead of updating the existing clones?

It's a different story of course if customers use the toolchain manager for installation, but then they wouldn't check this part of the docs anyway.

carlescufi commented:
> But we're in the installation instructions here - is it really a use case that customers install the NCS manually but don't clone the repos right after? Or that they clone the repos again for working on a different version instead of updating the existing clones?

I would say anyone who creates a second clone in their drive falls into that category, and we have many of those. For example, some people have a clone for master and another one for a particular release.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/nrfconnect/sdk-nrf?pullRequest=2312) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/nrfconnect/sdk-nrf?pullRequest=2312) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/nrfconnect/sdk-nrf?pullRequest=2312) it.</sub>

mbolivar-nordic commented on doc/nrf/gs_installing.rst:
```diff
@@ -191,6 +191,9 @@ Install the bootstrapper for west by entering the following command:
 
+
+.. _west_update:
+
 You only need to do this once.
 Like any other Python package, the west bootstrapper is updated regularly.
```
Probably want to rework the 'bootstrapper' part while you are here, since that no longer exists.

ru-fu replied:
Not sure what you mean here ... but I got rid of the word "bootstrapper", if that was all.

ru-fu commented:
Updated the cloning section a bit now and added the note in both locations.
Print-out: 
[Installing the nRF Connect SDK manually — nRF Connect SDK 1.2.pdf](https://github.com/nrfconnect/sdk-nrf/files/4682086/Installing.the.nRF.Connect.SDK.manually.nRF.Connect.SDK.1.2.pdf)
